### PR TITLE
Exposed admin copy as github action

### PR DESCRIPTION
**Pull request by Dee-pak**
Exposed admin copy as github action. Also added source and target environment for create in rolling-instance.yml

### CODE REVIEW

File: src/actions/copy-environment/index.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+import * as core from '@actions/core';
+import os = require('os');
```
Comment (davidjenni) [inline src/actions/copy-environment/index.ts:4]: please fix the lint warnings; it's a bug in the build systems that those don't fail the build

**Review by davidjenni — CHANGES_REQUESTED** [review]

File: src/actions/copy-environment/index.ts (hunk 1 above)
Comment (Dee-pak) [reply src/actions/copy-environment/index.ts]: Done.

File: src/actions/copy-environment/index.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+        const sourceUrl = core.getInput('source-url', { required: true});
+        const targetUrl = core.getInput('target-url', { required: true});
+
+        const pac = factory.getRunner('pac', process.cwd());
+        await pac.run(['auth', 'clear']);
+        await pac.run(['auth', 'create', '--kind', 'ADMIN', '--username', username, '--password', password]);
+
+        const copyEnvironmentArgs = ['admin', 'copy', '--source-url', sourceUrl, '--target-url', targetUrl];
+        const result = await pac.run(copyEnvironmentArgs);
```
Comment (davidjenni) [inline src/actions/copy-environment/index.ts:29]: I really need to fix lint to be a build error :-) here's another one

**Review by davidjenni — CHANGES_REQUESTED** [review]

File: src/actions/copy-environment/index.ts (hunk 2 above)
Comment (Dee-pak) [reply src/actions/copy-environment/index.ts:29]: I am sorry, I didn't see this one on my vscode. I'll fix it.

**Review by davidjenni — APPROVED** [review]

**Pull request merged**